docs/batches.md

```diff
@@ -357,3 +461,50 @@ Following is a sample proxy service that illustrates how to connect to Salesforc
    <description/>
   </proxy>
 ```
+2. Create a xml file called addBatch.xml containing the following xml:
+
+```xml
+<addBatch>
+   <apiVersion>34</apiVersion>
+   <accessToken>00D28000000erPd!ARsAQLkIRoO7KdP9dQeqEfZuEYmSBIw3.OqbHUZbFSDwDBEV9ginNZmNi4yJuf14bnLN2cVpkUqVyxpUl5gqZSqFV0v_90hf</accessToken>
+   <apiUrl>https://ap2.salesforce.com</apiUrl>
+   <refreshToken>5Aep861TSESvWeug_wh8Zdrl_XXXXXXXX.QEq4FGbXtI5ARrLxzibR</refreshToken>
+   <clientId>3MVG9ZL0ppGP5UrBKXXXXXXKwHHYHfEh.gTMriEXwhf6DFvyXXXXXXXwQxmFraW3k0KgU</clientId>
+   <clientSecret>914846950346786099</clientSecret>
+   <intervalTime>1000000</intervalTime>
+   <registryPath>connectors/SalesforceBulk</registryPath>
+   <contentType>text/csv</contentType>
+   <isQuery>false</isQuery>
+   <jobId>7500K00000GV8lBQAT</jobId>
+   <objects>
+      <values>Name,description
+        Tom Dameon,Created from Bulk API
+      </values>
+   </objects>
+</addBatch>                        
+```
+3. Replace the credentials with your values.
+
+4. Execute the following cURL command:
+
+```bash
+curl http://localhost:8280/services/salesforcebulk_addBatch -H "Content-Type: text/xml" -d @addBatch.xml
+```
+5. Salesforce returns a xml response as below.
```

Comment:
Salesforce returns an XML response similar to the one shown below:

Reply:
Fixed in a0448a990213d3cdb3f3ae158ef2307753bad900.